impeller/entity/contents/content_context.cc

```diff
@@ -157,6 +157,14 @@ ContentContext::ContentContext(std::shared_ptr<Context> context)
       CreateDefaultPipeline<LinearGradientFillPipeline>(*context_);
   radial_gradient_fill_pipelines_[{}] =
       CreateDefaultPipeline<RadialGradientFillPipeline>(*context_);
+#ifndef FML_OS_ANDROID
```

Comment:
I don't really love this pattern. I think we may eventually need better scaffolding for backend specific shaders